.coderabbit.yaml

```
@@ -0,0 +1,12 @@
+   # CodeRabbit配置文件
+   reviews:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 3

(indentation)